scripts/e2e-test.sh

```diff
@@ -3,7 +3,7 @@
 set -eux
 
 SCRIPTDIR=$(dirname "$(realpath "$0")")
-TESTS="install basic_volume_io node_disconnect/replica_pod_remove"
```

Comment:
Could you move the replica test to before the replica_pod_remove (which should be last, being the most disruptive)? 
I should have added a comment to that effect, but I've asked Blaise to add a comment to his PR.

Reply:
Ah, he didn't. Could you add a comment along the lines of "add new tests before replica_pod_remove" ?

Reply:
Done.